src/components/VideoTile.jsx

```diff
@@ -52,14 +55,14 @@ const Tile = ({ peerId, showStatsOnTiles, width, height }) => {
             />
           ) : null}
 
-          {track ? (
+          {track && !isAudioOnly ? (
```

Comment:
this might not turn off the incoming data. check in webrtc-internals once.

Reply:
it should, if the video was on the unmount useeffect will detach and if it's not on already, attach will never be called

Reply:
me and @raviteja83  checked webrtc-internal and we were still getting packets when video was unmounted, so we are now passing an extra `attach` prop to `Video`. Please check [this PR](https://github.com/100mslive/web-sdks/pull/322)

I've raised a PR against websdk repo. Closing this PR.